dags/max_ai/run_evaluation.py

```diff
@@ -0,0 +1,54 @@
+import dagster
+from django.conf import settings
+
+from dags.common import JobOwners
+from dags.max_ai.snapshot_project_data import (
+    snapshot_clickhouse_project_data,
+    snapshot_postgres_project_data,
+)
+
+
+def get_object_storage_endpoint() -> str:
+    if settings.DEBUG:
+        val = dagster.EnvVar("EVALS_DIND_OBJECT_STORAGE_ENDPOINT").get_value()
+        if not val:
+            raise ValueError("EVALS_DIND_OBJECT_STORAGE_ENDPOINT is not set")
+        return val
+    return settings.OBJECT_STORAGE_ENDPOINT
+
+
+class ExportProjectsConfig(dagster.Config):
+    project_ids: list[int]
+    """Project IDs to run the evaluation for."""
+
+
+@dagster.op(out=dagster.DynamicOut(int))
+def export_projects(config: ExportProjectsConfig):
+    seen_projects = set()
+    for pid in config.project_ids:
+        if pid in seen_projects:
+            continue
+        seen_projects.add(pid)
+        yield dagster.DynamicOutput(pid, mapping_key=str(pid))
+
+
+class EvaluationConfig(dagster.Config):
+    experiment_name: str
```

Comment:
Question: Is it defined for internal Dagster usage, as I can't properly find where/how it is used?